src/commands/fun/latex.ts

```diff
@@ -0,0 +1,82 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+const BACKGROUND_COLOR = `rgb(${255 - 47}, ${255 - 49}, ${255 - 54})`;
+const PADDING = 4;
+
+export default class extends Command {
+
+	// Made by Brendan Lewis (@craftablescience)
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
```

Comment:
You should probably defer the interaction right at the start. A network request and image processing could cause the response to take more than three seconds.